source/tutorials/write-delete-documents.txt

```diff
@@ -0,0 +1,186 @@
+class, construct a new instance of the class, then call one or more of its methods
+to modify the delete operation. You can chain these method calls together.
+To modify the behavior of the delete operation, pass the class instance and
+chained method calls as the second argument to the
+``deleteOne()`` or ``deleteMany()`` method.
+
+You can use the following methods in the ``DeleteOptions`` class
+to modify a delete method. All methods are optional.
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+   
+   * - Method
+     - Description
+
+   * - ``collation (Collation collation)``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the MongoDB Server manual.
+
+   * - ``hint (Bson hint)``
+     - | Gets or sets the index to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/delete/#std-label-deletes-array-hint>`
+         in the MongoDB Server manual.
+
+   * - ``hint (String hint)``
+     - | Gets or sets the index to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/delete/#std-label-deletes-array-hint>`
+         in the MongoDB Server manual.
+
+   * - ``let (Bson variables)``
+     - | A map of parameter names and values. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/delete/#std-label-delete-let-syntax>` in the
+         MongoDB Server manual.
+
+   * - ``comment (BsonValue comment)``
+     - | A comment to attach to the operation. For more information, see the :manual:`delete command
+         fields </reference/command/delete/#command-fields>` guide in the
+         MongoDB Server manual for more information.
+
+   * - ``comment (String comment)``
+     - | A comment to attach to the operation. For more information, see the :manual:`delete command
+         fields </reference/command/delete/#command-fields>` guide in the
+         MongoDB Server manual for more information.
+
+Example
+~~~~~~~
+
+The following code uses the ``deleteMany()`` method to delete all documents in
+the ``restaurants`` collection with a ``name`` value that includes the string ``"Mongo"``.
+It also uses the ``comment`` method to add a comment to the operation:
+
+.. code-block:: java
+   :copyable: true
+
+    ObservableSubscriber <DeleteResult> deleteManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
+    restaurants.deleteMany(regex("name", "Mongo"), new DeleteOptions().comment("Deleting Mongo restaurants")).subscribe(deleteManySubscriber);
+    deleteManySubscriber.await();
+
+.. Additional Information
+----------------------
+
+.. TODO:
+.. For runnable code examples of inserting documents with {+driver-short+}, see
+.. :ref: ADD HERE
```

Comment:
Is this an artifact of when you were creating the page or actually work to be done in later tickets? No problem leaving here if the latter.

Reply:
Actual work to be done in later tickets! This will be done after I have a page of runnable code examples.